docs/index.md

```diff
@@ -130,11 +127,11 @@ if __name__ == '__main__':
 	main()
 ```
 #### `From Main` Explained:
-**Using the [Hydra](https://hydra.cc/docs/intro/){:target="_blank"} library to manage configuration for your script.**
+**Using the [Hydra](https://hydra.cc/docs/intro/){:target="_blank"} library to manage configuration for your Core-Lib library.**
 
 #### Decorator: @hydra.main:
-- This decorator tells `Hydra` to use the specified `YAML` configuration file (`core_lib_config.yaml`) located in the current directory ('.') to configure your application. It's a convenient way to manage configurations for your script.
-- **`def main(cfg)`**: This is the main function of your script, which takes a `cfg` argument. This argument will hold the configuration provided by `Hydra`.
+- This decorator tells `Hydra` to use the specified `YAML` configuration file (`core_lib_config.yaml`) located in the current directory ('.') to configure your application. It's a convenient way to manage configurations for your Core-Lib library.
+- **`def main(cfg)`**: This is the main function of your Core-Lib library, which takes a `cfg` argument. This argument will hold the configuration provided by `Hydra`.
 #### Inside the main function:
   - **`your_core_lib = YourCoreLib(cfg)`**: This line initializes an instance of the `YourCoreLib` class (defined earlier) using the configuration `cfg` provided by `Hydra`. This means that your application will use the configuration parameters specified in `core_lib_config.yaml` to set up the `YourCoreLib` instance.
```

Comment:
missing hydra highlight bellow 

Defining a function `get_config()` to use Hydra for loading testing

Reply:
"- **Implementing the setUp method**:" 
needed be not as a new level. try to avoid 3 levels.